cloud/aws/services/ec2/service.go

```diff
@@ -17,6 +17,6 @@ package ec2
 // The interfaces are broken down like this to group functions together.
 // One alternative is to have a large list of functions from the ec2 client.
 type Service struct {
-	Instances instances
 	VPCs      vpcs
+	Instances instances
```

Comment:
Why the move here? This is probably going to get big and I was hoping to keep it alphabetical but I'm open to other ideas.

Reply:
No reason, it probably just slipped through. Will fix